### PR TITLE
WIP non-stop cody

### PR DESCRIPTION
**Pull request by abeatrix**
showdiff

Loom: https://www.loom.com/share/f5901196e7bc4cbf9d08131c6a3e0d3b

## Test plan

TBD

diff

Update on context menu:
image

### CODE REVIEW

File: client/cody/src/chat/ChatViewProvider.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ import { isError } from '@sourcegraph/cody-shared/src/utils'
 import { View } from '../../webviews/NavBar'
 import { LocalStorage } from '../command/LocalStorageProvider'
 import { updateConfiguration } from '../configuration'
+import { VSCodeEditor } from '../editor/vscode-editor'
```
Comment (toolmantim) [inline client/cody/src/chat/ChatViewProvider.ts:27]: Testing VSCode GitHub PR comments…

Comment (abeatrix) [reply client/cody/src/chat/ChatViewProvider.ts:27]: Nice! What I have here is just a draft I have some new updates I haven't pushed yet, will do it tomorrow 😁

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.09 kb) | 0.00% (+0.09 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/bee/fixupchat/compare-report.html;tab=live_object) for a full comparison between the commits 19098e7 and deb1a47 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Comment by abeatrix** [comment]
I tried to update decorations to indicates the job status for each code selection and got that to work on my local branch, but somehow broke the line tracker 😭 will try to fix it first thing next week but if I can't, at least the current branch works 😢

**Pull request closed without merging**